runtime/common/src/lib.rs

```diff
@@ -64,7 +64,8 @@ const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 pub const MAXIMUM_BLOCK_WEIGHT: Weight = 2 * WEIGHT_PER_SECOND;
 const_assert!(NORMAL_DISPATCH_RATIO.deconstruct() >= AVERAGE_ON_INITIALIZE_RATIO.deconstruct());
 parameter_types! {
-	pub const BlockHashCount: BlockNumber = 2400;
+	pub const BlockHashCountForCrab: BlockNumber = 256;
```

Comment:
why changing this?

Reply:
https://github.com/darwinia-network/darwinia/issues/680

Reply:
https://github.com/paritytech/frontier/pull/409

Reply:
「// move block hash pruning window by one block
		let block_hash_count = T::BlockHashCount::get();
		let to_remove = number.saturating_sub(block_hash_count).saturating_sub(One::one());」
- - - - - - - - - - - - - - -
When changing the block hash count from 1024 to 256, some dirty items might not be removed(be skipped) in system pallet finalize.